daffodil-lib/src/main/scala/org/apache/daffodil/cookers/EntityReplacer.scala

```diff
@@ -47,8 +47,7 @@ final class EntityReplacer {
   val dfdlCharClassEntityName = "NL|WSP|WSP\\*|WSP\\+|ES"
 
   val entityCharacterUnicode: List[(String, String, Matcher)] =
-    List(
-      ("NUL", "\u0000", Pattern.compile("%" + "NUL" + ";", Pattern.MULTILINE).matcher("")),
+    List(("NUL", "\u0000", Pattern.compile("%" + "NUL" + ";", Pattern.MULTILINE).matcher("")),
```

Comment:
Just eclipse changing the whitespace.... I will see if this can be avoided.

Reply:
Does not seem to be any way to avoid, other than turning off all auto-formatting.

Reply:
That's too bad. The other syntax is definitely the right thing to do here. You should probably look into using ``git add -p`` when adding changes to a commit. It's fine if Eclipse wants to make random changes, but that doesn't mean all those changes should be comitted, like this case. Using the ``-p`` flag will let you control exactly which hunks are added to a commit and you can ignore these wrong changes. Or you chould use ``git checkout -p`` to undo specific changes.